### PR TITLE
created SQL connector reproducer test for gh-3856

### PR DESCRIPTION
**Pull request by jsafarik**
//test: `@sql-connector`

### CODE REVIEW

File: ui-tests/src/test/resources/features/integrations/sql-connector.feature [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    And fill in values
+      | log level      | INFO  |
+      | Log Body       | true  |
+      | Log message Id | false |
+      | Log Headers    | false |
+      | Log everything | false |
+    Then click on the "Done" button
+
+    # select postgresDB as middle point
+    When add first step between START and STEP connection
```
Comment (asmigala) [inline ui-tests/src/test/resources/features/integrations/sql-connector.feature]: This looks like legacy step, please replace with  `add integration step on position "0"`

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/resources/features/integrations/sql-connector.feature (hunk 1 above)
Comment (jsafarik) [reply ui-tests/src/test/resources/features/integrations/sql-connector.feature]: updated

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**